src/storhaug

```diff
@@ -291,29 +305,44 @@ create_virt_ip()
     local ipcount=${1}; shift
     local ip=${1}; shift
 
-    pcs -f ${cibfile} resource create vip${ipcount} ocf:heartbeat:IPaddr2 \
-        params \
-            ip=${ip} \
-            flush_routes="true" \
-        op monitor interval=60s \
-        meta resource-stickiness="0"
+    if [[ "X${HA_SVC_NFS}X" == "XyesX" ]]; then
```

Comment:
There should only be one definition of the vipN resource, prior to this block, automatically part of a group (even if it's a group of only itself). This will remove the need for the `else` statement below and make it easier to add portblock for SMB.